compatibility/sandbox-migration/runner/Migration/KeyTransfer.hs

```diff
@@ -0,0 +1,157 @@
+-- Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+-- SPDX-License-Identifier: Apache-2.0
+
+{-# LANGUAGE DerivingStrategies #-}
+{-# LANGUAGE DuplicateRecordFields #-}
+module Migration.KeyTransfer (test) where
+
+import Control.Monad
+import Control.Monad.Except
+import qualified Data.Aeson as A
+import Data.List
+import qualified Data.Text as T
+import GHC.Generics (Generic)
+import System.IO.Extra
+import System.Process
+
+import Migration.Types
+
+diff :: Eq a => [a] -> [a] -> [a]
```

Comment:
I would suggest to rename this to `symDiff` to make it clear that this is the symmetric difference as opposed to just `\\` which is what people usually call difference.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/6298/commits/eb7e7a67bb6a1d84e4119388f3ed957722b1e3cd